applications/rag/frontend/README.md

```diff
@@ -3,98 +3,21 @@
 This directory contains the code for a frontend flask webserver integrating with the inference
 backend and (NYI) vector database through [LangChain](https://python.langchain.com/docs/get_started/introduction).
 
-## Cloud Run Setup
+The image is hosted on `us-central1-docker.pkg.dev/ai-on-gke/rag-on-gke` and used by the frontend deployment.
 
-1. Configure inference internal loadbalancer service:
-```
-export INFERENCE_DEPLOYMENT=mistral-7b-instruct  # Customize as needed
-kubectl create service loadbalancer $INFERENCE_DEPLOYMENT --tcp=80:8080 && \
-kubectl annotate service/$INFERENCE_DEPLOYMENT "cloud.google.com/load-balancer-type=Internal"
+## Sensitive Data Protection
+Data Loss Prevention (DLP) and Natural Language Protection (NLP) System. Our integrated solution leverages the latest in data protection and language understanding technologies to safeguard sensitive information and ensure content appropriateness across digital platforms.
 
-# Record the LoadBalancer IP
-kubectl get service $INFERENCE_DEPLOYMENT --watch # Wait for external endpoint to be populated, then ^C
-export INFERENCE_ENDPOINT=$(kubectl get svc mistral-7b-instruct -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
-```
+### Features:
 
-2. Configure environment variables
-```
-# Change these values to match your desired Cloud Run deployment
-export PROJECT_ID=my-project
-export PROJECT_NUM=$(gcloud projects describe $PROJECT_ID --format="value(projectNumber)")
-export REGION=us-central1
+#### Data Loss Prevention (DLP): 
+Identifies, monitors, and protects sensitive data through detection techniques, ensuring compliance with data protection regulations. 
```

Comment:
Rewrite this specific to how we've implemented DLP in the demo. For example, note that it redacts potentially sensitive information from outputs (and data if we get to implementing that). We don't currently support any compliance regimes with our implementation here.